sea-query-binder/src/values.rs

```diff
@@ -1,2 +1,2 @@
 #[derive(Clone, Debug, PartialEq)]
-pub struct SqlxValues(pub(crate) sea_query::Values);
+pub struct SqlxValues(pub sea_query::Values);
```

Comment:
I guess this isn't a must?

Reply:
This is needed for SeaORM: https://github.com/SeaQL/sea-orm/pull/970.

Reply:
I see! That make more sense now. Thanks!!